plugins/lookup/tss.py

```diff
@@ -211,13 +241,28 @@ def from_params(**server_parameters):
         else:
             return TSSClientV0(**server_parameters)
 
-    def get_secret(self, term):
+    def get_secret(self, term, fetch_file_attachments, file_download_path):
         display.debug("tss_lookup term: %s" % term)
-
         secret_id = self._term_to_secret_id(term)
         display.vvv(u"Secret Server lookup of Secret with ID %d" % secret_id)
 
-        return self._client.get_secret_json(secret_id)
+        if fetch_file_attachments:
+            obj = self._client.get_secret(secret_id, fetch_file_attachments)
+            for i in obj['items']:
+                if os.path.isdir(file_download_path):
+                    if i['isFile']:
+                        try:
+                            f = open(os.path.join(file_download_path, str(obj['id']) + "_" + i['slug']), "w")
+                            f.write(i['itemValue'].text)
+                            f.close()
```

Comment:
Please use modern Python syntax:
```suggestion
                            with open(os.path.join(file_download_path, str(obj['id']) + "_" + i['slug']), "w") as f:
                                f.write(i['itemValue'].text)
```
Also note that you are writing the file with an unknown encoding. To make this work both with Python 2 and 3, better use `io.open()` instead of `open()` and specify `encoding`.